packages/api/src/providers/oauth1.ts

```diff
@@ -29,7 +29,7 @@ export interface OAuth1 {
     userAccessToken: string;
     userAccessTokenSecret: string;
   }>;
-  deregister(userAccessTokens: string[]): Promise<void>;
+  deregister(userAccessTokens: string[], cxId?: string): Promise<void>;
```

Comment:
I see we're adding the `cxId` to `deregister` - I think b/c we want to de-register to a specific cx, but I couldn't find a place that's calling OAuth1's `deregister` w/ `cxId`. Maybe missed adding it when calling `deregister([token]);`?

Also, we are not differentiating the request to de-register a user coming from the Provider (de-register all cxs) vs coming from our cx (de-register on that one cx), right? Shouldn't we?